src/main/java/life/genny/qwandaq/constant/MinIOConstant.java

```diff
@@ -0,0 +1,31 @@
+package life.genny.qwandaq.constant;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public class MinIOConstant {
+
+    public static String getServerUrl(){
+        return System.getenv("MINIO_SERVER_URL") != null
+                ? System.getenv("MINIO_SERVER_URL")
```

Comment:
Extract to constants

Reply:
Since static variable got initialized before the System Environment loaded, we got a null value so we used getters to fix this issue.